bot/seasons/evergreen/recommend_game.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+from pathlib import Path
+from random import shuffle
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+DIR = Path("bot/resources/evergreen/game_recs")
+game_recs = []
+
+# Populate the list `game_recs` with resource files
+for file_url in DIR.glob("*.json"):
+    with Path(file_url).open(encoding='utf-8') as file:
+        data = json.load(file)
+    game_recs.append(data)
+shuffle(game_recs)
+
+
+class RecommendGame(commands.Cog):
+    """Commands related to recommending games."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.index = 0
+
+    @commands.command(name="recommend_game", aliases=['gameRec', 'recommendGame'])
```

Comment:
From what I've seen, none of the other commands use an underscore or camel case for multiple words. Anyone please correct me if I am wrong or you otherwise disagree.

```suggestion
    @commands.command(name="recommendgame", aliases=['gamerec'])
```